JULIERAJ/KIDS-FIRST-Front
# Fix/40/register design and error handling

## Register design and error handling

## Requirements
- Register form should be refactored
- Password requirements should be updated while typing
- Error cases should be matched
- Error when register should be under password
- Layout should be scrollale to match Figma + when errors occur there should be extra space under inputs

## Test Steps
- Try to register without an email or log in. Or without first name. Or password. Try also to sign in without an email or password, try to register with the existing email. As a result, you should see relevant messages.
- Check requirements change while typing a password.
- Check UI/UX Outcome

## UI/UX Outcome
<img width="1691" alt="Screenshot 2024-07-17 at 17 15 26" src="https://github.com/user-attachments/assets/ff1a2099-5453-4496-8531-8cf13078a9fc">

## Checklist
